### PR TITLE
add quarkiverse-cx to quarkus-ci

### PR DESCRIPTION
**Pull request by dufoli**
Related issue: https://github.com/quarkiverse/quarkiverse/issues/14

### CODE REVIEW

**Comment by Dufgui** [comment]
I think you forget to create the issue on quarkus CI no?

**Comment by dufoli** [comment]
@Dufgui just fix it

File: quarkiverse-cxf/info.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+url: https://github.com/quarkiverse/quarkiverse-cxf
+issues:
+  repo: quarkiverse/quarkiverse
+  latestCommit: 13046
```
Comment (gastaldi) [inline quarkiverse-cxf/info.yaml:4]: This must be an issue created in the quarkiverse/quarkiverse repository

**Review by gastaldi — CHANGES_REQUESTED** [review]

File: quarkiverse-cxf/info.yaml (hunk 1 above)
Comment (dufoli) [reply quarkiverse-cxf/info.yaml]: ok done ;-)

**Comment by dufoli** [comment]
my issue now is that I have not access to setting of quarkiverse-cxf only to team.

**Pull request merged**